esmvaltool/recipes/recipe_li17natcc.yml

```diff
@@ -0,0 +1,192 @@
+# recipe_li17natcc.yml
+---
+documentation:
+  description: |
+    Recipe for reproducing the plots in Li et al., 2017 Nature climate change
+
+  authors:
+    - weigel_katja
+
+  references:
+    - li17natcc
+
+  projects:
+    - eval4cmip
+
+preprocessors:
+  pp850:
+    extract_levels:
+      levels: 85000
+      scheme: linear
+    regrid:
+      target_grid: 2x2
+      scheme: linear
+    extract_region:
+      start_latitude: -40
+      end_latitude: 40
+      start_longitude: 0
+      end_longitude: 360
+  tropical:
+    regrid:
+      target_grid: 2x2
+      scheme: linear
+    extract_region:
+      start_latitude: -40
+      end_latitude: 40
+      start_longitude: 0
+      end_longitude: 360
+  glob:
+    mask_landsea:
+      mask_out: land
+    area_statistics:
+      operator: mean
+
+diagnostics:
+
+  cmip5:
+    description: Diagnostic for figure 1 and 2 from Li et al., 2017. Data sets from CMIP5.
+    themes:
+      - EC
+    realms:
+      - atmos
+    variables:
+      pr: &gridded_cmip5_r1i1p1_amon_t2ms
+        preprocessor: tropical
+        project: CMIP5
+        ensemble: r1i1p1
+        mip: Amon
+        field: T2Ms
```

Comment:
Is the 'field' tag needed for CMIP data?

Reply:
Not anymore I guess?

Reply:
It works without the tag field, therefore I removed it.